### PR TITLE
IDEA-188241 Hide AS icons for not Android projects

### PR DESCRIPTION
**Pull request by dahlstrom-g**
This revises and expands on commit 66c0b98.

### CODE REVIEW

**Comment by dahlstrom-g** [comment]
@dmitry-avdeev?

File: profilers-android/src/com/android/tools/idea/profilers/ProfileRunExecutor.java [hunk 1]
```diff
@@ -95,6 +97,9 @@ public Icon getExecutorIcon(@NotNull Project project, @NotNull Executor executor
 
   @Override
   public boolean isApplicable(@NotNull Project project) {
-    return !ProjectFacetManager.getInstance(project).getFacets(AndroidFacet.ID).isEmpty();
+    ProjectFacetManager facetManager = ProjectFacetManager.getInstance(project);
+    return facetManager.hasFacets(AndroidFacet.getFacetType().getId()) ||
+           facetManager.hasFacets(ApkFacet.getFacetTypeId()) ||
```
Comment (dmitry-avdeev) [inline profilers-android/src/com/android/tools/idea/profilers/ProfileRunExecutor.java:102]: Are you sure any Gradle project can be profiled with Android profiler?

Comment (dahlstrom-g) [reply profilers-android/src/com/android/tools/idea/profilers/ProfileRunExecutor.java:102]: Android Gradle projects, yes; they're the kind Android Studio normally uses. Did you look at GradleFacet?

Comment (dmitry-avdeev) [reply profilers-android/src/com/android/tools/idea/profilers/ProfileRunExecutor.java:102]: Ok, I see it now. GradleFacet name was a bit misleading.

Comment (dahlstrom-g) [reply profilers-android/src/com/android/tools/idea/profilers/ProfileRunExecutor.java:102]: I agree! Thanks for taking a look.

**Comment by dahlstrom-g** [comment]
@dmitry-avdeev Is there anything in the way of merging this?

**Comment by dmitry-avdeev** [comment]
Merged, thank you!

**Pull request closed without merging**